### PR TITLE
Refine docs — project initialization

### PR DESCRIPTION
**Pull request by kasperski95**
Related https://github.com/software-mansion/protostar/issues/1194

Feel free to merge if approved.

### CODE REVIEW

File: website/docs/tutorials/03-project-initialization.md [hunk 1]
```diff
@@ -4,21 +4,33 @@ sidebar_label: Project initialization
...
+your-project-name/
+├── src/
+│   └── main.cairo
+├── tests/
+│   └── test_main.cairo
+└── protostar.toml
+```
+
+The example contract file, `main.cairo`, and the example test file, `test_main.cairo`, serve as templates for your project.
+You can modify these files or add additional files as needed to meet the requirements of your project.
```
Comment (Arcticae) [inline website/docs/tutorials/03-project-initialization.md:26]:
```suggestion
```

File: website/docs/tutorials/03-project-initialization.md [hunk 2]
```diff
@@ -4,21 +4,33 @@ sidebar_label: Project initialization
...
+
+The configuration file, `protostar.toml`, defines the root directory of your Protostar project.
 
-The result of running `protostar init` is a configuration file `protostar.toml`, example files, and the following 3 directories:
+### Adapting an existing project to a Protostar project
+If you already have an existing project that you want to adapt to a Protostar project, you can do so by adding a valid `protostar.toml` configuration file to the root of the project.
```
Comment (Arcticae) [inline website/docs/tutorials/03-project-initialization.md:31]:
```suggestion
If you already have an existing project that you want to work with Protostar, you can do so by adding a valid `protostar.toml` configuration file to the root of the project.
```

File: website/docs/tutorials/03-project-initialization.md [hunk 3]
```diff
@@ -4,21 +4,33 @@ sidebar_label: Project initialization
...
 
-- `src` — A directory for your code.
-- `lib` — A default directory for an external dependencies.
-- `tests` — A directory storing tests.
+Alternatively, you can use the `protostar init --existing` command to create a `protostar.toml` in your existing project.
+This command will create the `protostar.toml` file in the current directory, which should be the root of your existing project.
```
Comment (Arcticae) [inline website/docs/tutorials/03-project-initialization.md:34]:
```suggestion
Alternatively, you can use the `protostar init --existing` command to generate a `protostar.toml` in your current working directory.
```

**Review by Arcticae — APPROVED** [review]
LGTM, suggesting some simplifications

File: website/docs/tutorials/03-project-initialization.md (hunk 2 above)
Comment (kasperski95) [reply website/docs/tutorials/03-project-initialization.md:30]: I think both versions are correct. Leaving the original one.

**Pull request merged**